### PR TITLE
Updated the TestStand sequence of measurement examples to make it simpler (DMM, FGEN, SCOPE, SWITCH, VISA DMM)

### PR DESCRIPTION
**Pull request by vigkre**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?
- Updates the TestStand sequence in the example measurement to make it simpler by moving the process setup and process cleanup steps to the main sequence.
-  The sequence update and their relevant python functions are done for below examples:
     - DMM
     - FGEN
     - SCOPE
     - SWITCH
     - VISA DMM
- Combines the logic for registering pin map into a single step in the Setup section. Previously this was handled in three steps.
- Updated the `Readme.md` file for the updated examples.

### Why should this Pull Request be merged?
Based on the feedback discussed in this [Teams post](https://teams.microsoft.com/l/message/19:36f0e963feb84c0c8122e6ddfbe7df7b@thread.tacv2/1676494204834?tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e&groupId=a092c504-be1c-4d21-9ef4-068a40ca6974&parentMessageId=1676494204834&teamName=MeasurementLink%20%26%20ETW%20Global%20Collaboration&channelName=Roadmap%20Input%20and%20Feature%20Ideas&createdTime=1676494204834&allowXTenantAccess=false), moving the process setup and process cleanup steps to the main sequence will simplify the example sequence as the user will only need to run the main sequence to get the expected result. Though this sequence is not efficient for multi-site use case, this change is encouraged as the example sequence is expected to be tailored to users who are beginners in TestStand.

### What testing has been done?
- Manually executed the TestStand example sequence with a simulated instrument and verified the result for below examples
     - DMM
     - FGEN
     - SCOPE
     - SWITCH
     - VISA DMM

### CODE REVIEW

File: examples/niscope_acquire_waveform/NIScopeAcquireWaveform.seq [hunk 1]
```diff
@@ -4588,16 +4588,16 @@
 																					<value>pin_names</value>
 																				</Name>
 																				<ArgumentValue classname='ExprValue'>
-																					<value>"Pin1,Pin2,Pin3,Pin4"</value>
+																					<value>{"Pin1","Pin2","Pin3","Pin4"}</value>
```
Comment (bkeryan) [inline examples/niscope_acquire_waveform/NIScopeAcquireWaveform.seq:4591]: Did this have the wrong data type before?

File: examples/nidmm_measurement/NIDmmMeasurement.seq
Comment (bkeryan) [inline examples/nidmm_measurement/NIDmmMeasurement.seq:1]: These steps have been moved into the code module. Delete them.

image

Comment (bkeryan) [inline examples/nidmm_measurement/NIDmmMeasurement.seq:1]: Delete `PinMapFileName`. Set `PinMapPath` to the relative path. Make sure to pass `PinMapPath` to `update_pin_map`.

image

File: examples/nidmm_measurement/_helpers.py
Comment (bkeryan) [inline examples/nidmm_measurement/_helpers.py:1]: `_helpers.py` should be identical in each example. `nivisa_dmm_measurement` still has the old `_helpers.py`.

This highlights the fact that this PR doesn't update `nivisa_dmm_measurement`'s TestStand support.

```
D:\dev\measurementlink-python>dir /s _helpers.py

 Directory of D:\dev\measurementlink-python\examples\nidcpower_source_dc_voltage

04/29/2023  06:15 PM             6,303 _helpers.py
               1 File(s)          6,303 bytes

 Directory of D:\dev\measurementlink-python\examples\nidmm_measurement

04/29/2023  07:02 PM             6,303 _helpers.py
               1 File(s)          6,303 bytes

 Directory of D:\dev\measurementlink-python\examples\nifgen_standard_function

04/29/2023  07:02 PM             6,303 _helpers.py
               1 File(s)          6,303 bytes

 Directory of D:\dev\measurementlink-python\examples\niscope_acquire_waveform

04/29/2023  07:02 PM             6,303 _helpers.py
               1 File(s)          6,303 bytes

 Directory of D:\dev\measurementlink-python\examples\niswitch_control_relays

04/29/2023  07:02 PM             6,303 _helpers.py
               1 File(s)          6,303 bytes

 Directory of D:\dev\measurementlink-python\examples\nivisa_dmm_measurement

03/16/2023  02:27 PM             3,903 _helpers.py
               1 File(s)          3,903 bytes
```

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: examples/nidmm_measurement/NIDmmMeasurement.seq
Comment (vigkre) [reply examples/nidmm_measurement/NIDmmMeasurement.seq:1]: Deleted the steps which was moved into code module.

Comment (vigkre) [reply examples/nidmm_measurement/NIDmmMeasurement.seq:1]: Deleted `PinMapFileName` and `PinMapPath` has been set to relative path. `PinMapPath` variable was already passed into `update_pin_map` function

File: examples/nidmm_measurement/_helpers.py
Comment (vigkre) [reply examples/nidmm_measurement/_helpers.py:1]: Yes Brad. In this PR, we are covering other examples. `nivisa_dmm_measurement` not yet updated and will be handled in a separate PR

File: examples/niscope_acquire_waveform/NIScopeAcquireWaveform.seq (hunk 1 above)
Comment (vigkre) [reply examples/niscope_acquire_waveform/NIScopeAcquireWaveform.seq]: Yes Brad.  Adding the screenshot of previous version

Initial load of seq file:

image


After Reloading measurement params:

image

File: examples/nidmm_measurement/_helpers.py
Comment (vigkre) [reply examples/nidmm_measurement/_helpers.py:1]: We initially thought that VISA DMM device simulation was different from other examples. So, thought of handling it and create a separate PR.

Now, after taking a look into the simulation, it seemed like simple. So, I have updated the sequence file and its related functions updates in `_helpers.py` and `teststand_fixture.py`.

File: examples/nidcpower_source_dc_voltage/README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
   `ui_file_paths` array in `measurement.py`
 - Includes a TestStand sequence showing how to configure the pin map, register
   instrument sessions with the session management service, and run a measurement
+  - For the sake of simplicity, the TestStand sequence handles pin map and session registration and unregistration in the `Setup` and `Cleanup` sections of the main sequence. For **Test UUTs** and batch process model use cases, these steps should be moved to the `ProcessSetup` and `ProcessCleanup` callbacks.
```
Comment (bkeryan) [inline examples/nidcpower_source_dc_voltage/README.md:19]: Style: the rest of these READMEs are word wrapped

Comment (vigkre) [reply examples/nidcpower_source_dc_voltage/README.md]: Word wrapped for the newly added README.

**Pull request merged**